portfolio/src/main/webapp/commentsection.js

```diff
@@ -52,25 +94,34 @@ function createComment(commentData) {
         val = Math.floor(commentData.time/(1000*60*60*24));
```

Comment:
The "else" branch contains an error. Instead of "1000*60*60*24" you should use "1000*60*60*24*365".